_posts/2024-03-11-java-22.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: "Blazing ahead with 22"
+author: "Manfred Moser, Martin Traverso, Dain Sundstrom, David Phillips"
+excerpt_separator: <!--more-->
+image: /assets/images/logos/java-duke-22.png
+show_pagenav: true
+---
+
+It really has not been that long ago that we [first announced support for Java
+21]({% post_url 2023-11-03-java-21 %}), and subsequently made is a build and
+runtime requirement with [Trino 436]({{site.url}}docs/current/release/release-436.html). 
+
+Since then the codebase received some significant improvements in readability,
+and we have also seen better performance. However, innovation in Trino and Java
+is not holding still, on the contrary its accelerating. On the Java commmunity
+side, Java 22 is just about to be released, and we think it is time to drive
+innovation in Trino even further.
```

Comment:
I think in general, you should say in your intro paragraphs very explicitly that Trino is moving to Java 22.

Reply:
Done